Engine/platform/osx/acplmac.cpp

```diff
@@ -113,6 +151,31 @@ void AGSMac::ShutdownCDPlayer() {
   //cd_exit();
 }
 
+const char* AGSMac::GetAllUsersDataDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetUserSavedgamesDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetUserConfigDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetAppOutputDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetIllegalFileChars()
+{
+  return "\\/:?\"<>|*"; // keep same as Windows so we can sync.
```

Comment:
No, actually, since the rule for portable names is supposed to be always same, then such function should not be overriden in a platform class at all!